lib/fs.js

```diff
@@ -17,6 +18,20 @@ const mkdir = (dir, mode, cb) => {
   });
 };
 
+const recursivlyListDirs = dir => {
```

Comment:
It has incorrect behaviour. It should list directories which are nested in `dir`, not vice versa.